hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestDataNodeReconfiguration.java

```diff
@@ -365,4 +372,79 @@ public void testBlockReportIntervalReconfiguration()
           .getConf().get(DFS_BLOCKREPORT_INTERVAL_MSEC_KEY));
     }
   }
+
+  @Test
+  public void testSlowPeerParameters()
+      throws ReconfigurationException {
+    String[] slowPeersParameters = {
+        DFS_DATANODE_MIN_OUTLIER_DETECTION_NODES_KEY,
+        DFS_DATANODE_SLOWPEER_LOW_THRESHOLD_MS_KEY,
+        DFS_DATANODE_PEER_METRICS_MIN_OUTLIER_DETECTION_SAMPLES_KEY};
+
+    for (int i = 0; i < NUM_DATA_NODE; i++) {
+      DataNode dn = cluster.getDataNodes().get(i);
+
+      // Try invalid values.
+      try {
+        dn.reconfigureProperty(DFS_DATANODE_PEER_STATS_ENABLED_KEY, "text");
+      } catch (ReconfigurationException expected) {
+        assertEquals("Could not change property dfs.datanode.peer.stats.enabled from 'true' to " +
+            "'text'", expected.getMessage());
+      }
```

Comment:
Use `LambdaTestUtils` instead for such cases

Reply:
Good idea.

Reply:
Hi @ayushtkn , I update it, PTAL. Thanks.